src/status_im2/constants.cljs

```diff
@@ -39,6 +39,10 @@
 (def ^:const activity-center-mark-all-as-read-undo-time-limit-ms 4000)
 (def ^:const activity-center-max-unread-count 99)
 
+;; (def ^:const activity-center-community-request-timeout-ms (* 7 24 60 60 1000))
+;; For testing we set request timeout to 5 minutes
+(def ^:const activity-center-community-request-timeout-ms (* 5 60 1000))
```

Comment:
Maybe use environment variables so for test builds timeout is 5 minutes and for production build is 7 days. wdyt?

Reply:
The 7 days timeout is handled in `status-go` (https://github.com/status-im/status-go/pull/3387/files#diff-702ae791906d65cbc687cc00b48d95a27f50d11aff4f1ec1fa47335777e37a63). I think with the current implementation in `status-go`, we can not know the type of build. We need to implement it if we need it.

If the request goes unattended or is declined, after the timeout, the status-go will delete the `request` from the sender and admin and also deletes the "Join request" notification from the admin's Activity Centre. 

But for the sender/requester, this "Requested to join <COMMUNITY>" notification would persist. During the rendering of this notification, we need to check the request time for timeout (with help of this constant) and render the text appropriately.

The QA team requested 5 minutes to test this PR/feature. I will remove these 5 minutes (on mobile and status-go) once the testing is completed.